Formula/visp.rb

```diff
@@ -53,7 +51,8 @@ def install
              "C Compiler:                  #{ENV.cc}\"")
     end
 
-    system "cmake", ".", "-DBUILD_DEMOS=OFF",
+    system "cmake", ".", "-DUSE_CXX_STANDARD=11",
```

Comment:
This actually doesn't work. It looks like some dependency headers need newer standard (e.g. C++14). By not setting this, I see build ends up adding `-std=c++17`.